env0/resource_template_test.go

```diff
@@ -146,7 +146,48 @@ func TestUnitTemplateResource(t *testing.T) {
 		GithubInstallationId: 2,
 		TerraformVersion:     "0.15.1",
 	}
-
+	bitbucketTemplate := client.Template{
```

Comment:
also, update the `mixedUsecases` matrix it checks you cannot set multiple vcs for one template

Reply:
added.